js/utils.js

```diff
@@ -1081,11 +1081,11 @@ Fliplet.Registry.set('dynamicListUtils', (function () {
 
           return 0;
         case 'number':
-          if (!parseInt(aValue, 10)) {
+          if (+aValue !== 0 && !parseFloat(aValue, 10)) {
```

Comment:
Please explain your fix here. And, why do we need to use `+aValue` when we have `parseFloat()` already?

Reply:
If `aValue` is equal to 0 result of `parseFloat(0, 10)` would be 0 and `Boolean(0)` will return `false` but 0 is a number and should be compared with other numbers greater or less than 0. Otherwise it will be sorted wrong:
![image](https://user-images.githubusercontent.com/74735207/102099506-06014c00-3e31-11eb-9242-b5115fcfff1d.png)

Reply:
Thanks for the explanation. Makes sense now.